.azure-pipelines/testbed_health_check.py

```diff
@@ -111,6 +113,10 @@ def init_hosts(self):
         if not self.sonichosts:
             raise HostInitFailed("sonichosts is None. Please check testbed name/file/inventory.")
 
+        self.duts_basic_facts = self.sonichosts.dut_basic_facts()
+        self.is_multi_asic = self.duts_basic_facts[self.sonichosts[0].hostname][
+            "ansible_facts"]["dut_basic_facts"]["is_multi_asic"]
+
```

Comment:
If we already defined `is_multi_asic`, we can enhance the code to skip multi-asic check at the end of pre-check:
```
        # Retrieve the basic facts of the DUTs
        duts_basic_facts = self.sonichosts.dut_basic_facts()

        for dut_name, single_dut_basic_facts in duts_basic_facts.items():

            # Get the basic facts of one DUT
            dut_basic_facts = single_dut_basic_facts["ansible_facts"]["dut_basic_facts"]

            # todo: Skip multi_asic check on multi_asic dut now because currently not support get asic object
            if dut_basic_facts["is_multi_asic"]:
                errmsg = "Not support to perform checks on multi-asic DUT now."
                logger.info(errmsg)

                raise SkipCurrentTestbed(errmsg)
```